src/components/Login.js

```diff
@@ -31,14 +29,27 @@ const Login = () => {
 
     const submitLogin = (auth) => {
         axios
-            .post('http://localhost:4000/auth', auth)
-            .then(() => {
-                window.location.href="/session";},
-                (res) => setResponse('ERROR: User not authenticated.'))
+            .post('http://localhost:4000/auth', auth, {withCredentials: true})
+            .then((x) => {
+                console.log(x);
+                console.log('Routing to /session from React')
+                // window.location.href="/session"
+            })
+    }
+
+    const authTest = (auth) => {
```

Comment:
Consider abstraction

```
const API_URL = "http://localhost:4000"
```

When you deploy this to staging/prod you will need to configure server URL